### DISCUSSION TITLE
Self supervised SwinUnetr Pretrained weight usage

### DISCUSSION THREAD

**Discussion by justinzhang528** [Project-MONAI/tutorials#1382] [Q&A] [answered]
Dear MONAI Community,

I have a question about the usage of [swinUnetr pretrained weight](https://github.com/Project-MONAI/tutorials/tree/main/self_supervised_pretraining/swinunetr_pretrained), if i want to use the pretrained weight for my down stream task, is any constrain about input image such as image size, channel or dimension...

Thank in advance!

**Comment by KumoLiu** [comment, marked as answer]
Hi @justinzhang528, I think you can find more details in this [tutorial](https://github.com/Project-MONAI/tutorials/blob/main/self_supervised_pretraining/swinunetr_pretrained/swinunetr_finetune.ipynb) about how to use this pre-trained weight.
```
model = SwinUNETR(
    img_size=(96, 96, 96),
    in_channels=1,
    out_channels=14,
    feature_size=48,
    drop_rate=0.0,
    attn_drop_rate=0.0,
    dropout_path_rate=0.0,
    use_checkpoint=True,
)
```
Hope it can help you, thanks!

**Reply by justinzhang528** [reply]
@KumoLiu I will take a look, thanks！

**Comment by tangy5** [comment]
If you would like to use the pretrained position embedding, you will need to follow the 96x96x96 patch size as input. 
If you'd like to load part of the pre-trained model, for example, only the transformer blocks, it should be good to variant image size inputs. But you will need to customize the weights loading logics. 

Thank you.

**Reply by justinzhang528** [reply]
I got it, thanks!
